sentry_sdk/integrations/starlette.py

```diff
@@ -339,7 +339,13 @@ async def form(self):
         curl -X POST http://localhost:8000/upload/somethign -H "Content-Type: application/x-www-form-urlencoded" -d "username=kevin&password=welcome123"
         curl -X POST http://localhost:8000/upload/somethign  -F username=Julian -F password=hello123
         """
-        return await self.request.form()
+        try:
```

Comment:
in general in lots of other places, we try to import stuff and fail there, but up to you however you prefer to do it
https://github.com/getsentry/sentry-python/blob/9e69fcf61a537ae30ea687f5b55957b6f3441935/sentry_sdk/integrations/flask.py#L23-L26

Reply:
Ok. Done.